content/en/docs/releases/security-vulnerabilities/index.md

```diff
@@ -87,3 +88,18 @@ branches.
 As much as possible this announcement will be actionable, and include any mitigating steps customers can take prior to
 upgrading to a fixed version. The recommended target time for these announcements is 16:00 UTC from Monday to Thursday.
 This means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.
+
+## Base Images
+
+Istio offers two sets of docker images, based on `ubuntu` (default) and based on `distroless` (see [Harden Docker Container Images](docs/ops/configuration/security/harden-docker-images/)).
+These base images occasionally have CVEs.
+The Istio security team has automated scanning to ensure base images are kept free of CVEs.
```

Comment:
product security?

Reply:
rest of page uses "Istio security team". Want me to change them all?

Reply:
Nope. Sounds like that's been the norm, and I had missed that.